frontend/app/components/input/input.tsx

```diff
@@ -135,6 +136,13 @@ export class Input extends Component<Props, State> {
     });
   }
 
+  onPaste(e: ClipboardEvent) {
+    if (e.clipboardData && e.clipboardData.files.length > 0) {
+      const files = (e.clipboardData.files as unknown) as File[];
+      this.uploadImages(files);
```

Comment:
You should use `e.preventDefault` or else code works incorrectly. Attempting to upload file with name `Screenshot 2019-07-15 at 23.40.48.png` results in `![image.png](http://127.0.0.1:9000/api/v1/picture/dev_user/bkmenb6q9r8cnr0sfqhg.png).40.48.png` being inserted in input.

```
  onPaste(e: ClipboardEvent) {
    if (!e.clipboardData || e.clipboardData.files.length === 0) return;

    e.preventDefault();
    const files = Array.from(e.clipboardData.files);
    this.uploadImages(files);
  }
```

Reply:
hmm, found out, that if you're trying to paste more that one image, then pasting doesn't work because clipboardData.files.length === 0 in that case. Even on github. Strange :-)

Reply:
It is because clipboard data can contains different object types, this behaviour depend on  OS and browser

Reply:
Yeah, I suspect it's Firefox issue